### DISCUSSION TITLE
How to add styles to an element "trigger" - popoverTriggerClass

### DISCUSSION THREAD

**Discussion by magistr4815** [Akryum/v-tooltip#424] [Ideas]
Hey. You wrote a very good component.
But why there is no props "**popoverTriggerClass**", so that I can add my custom styles for the trigger element.

Code Popover.vue
```
<div
      ref="trigger"
      class="trigger"
      // my modification
      // :class="['trigger', popoverTriggerClass]"
      style="display: inline-block;"
      :aria-describedby="popoverId"
      :tabindex="trigger.indexOf('focus') !== -1 ? 0 : undefined"
></div>
```

**Comment by misha** [comment]
Just ran into this as well. I was able to select the element via the `trigger` CSS class in a global CSS block, ie. not `scoped`, but it would be great to allow custom classes on that element.
